src/activity-vertical-and-horizontal-translations.xml

```diff
@@ -225,76 +220,83 @@
             $input=Compute("t-3");
             $addOrSubtract=PopUp(["?","add","subtract"],"add");
             $outputChange=Real(12);
-
           </pg-code>
 
         </setup>
 
         <statement>
-
           <p>Again, the function <m>f(t) = -4.9t^2 + 10t + 40</m> represents the height of an object which was thrown straight upward at a speed of 10 meters per second, from an initial height of 40 meters.</p>
           <p>Suppose now that the person waited <m>3</m> seconds and also carried it an additional 12 meters higher in elevation before throwing it.</p>
           <p>Waiting 3 seconds is a change to the input <m>t</m>, so the input of the function will now be: <var name="$input" width="5" /></p>
           <p>Carrying the object an additional 12 meters upward before throwing it is a change to the output, so we will <var name="$addOrSubtract" form="popup" /> the number <m>12</m> to the outside of the function.</p>
           <p>Putting these transformations together, the formula will now be:</p>
-          <p><m>f(</m><var name="$input" width="5" /><m>) + </m><var name="$outputChange" width="2" /></p>
-          <p>= <m>-4.9(</m><var name="$input" width="5" /><m>)^2 + 10(</m><var name="$input" width="5" /><m>) + 40 + </m><var name="$outputChange" width="2" /></p>
+          <p><m>f\big(</m><var name="$input" width="5" /><m>\big) + </m><var name="$outputChange" width="2" /></p>
+          <p>= <m>-4.9\big(</m><var name="$input" width="5" /><m>\big)^2 + 10\big(</m><var name="$input" width="5" /><m>\big) + 40 + </m><var name="$outputChange" width="2" /></p>
```

Comment:
I'd put the = sign in with the math environment to make it look nicer and have better spacing. If you precede the = sign with `{}` then you'll get a little bump of spacing to the left of the = sign too (once it is in the math tag).